### DISCUSSION TITLE
Cloudflare Integration

### DISCUSSION THREAD

**Discussion by SingularDave** [arthelokyo/astrowind#157] [Q&A] [answered]
Hi there! I'm a newcomer to Astro and I'd like to use this theme along with Cloudflare integration. However, I'm unable to utilize the new experimental Images feature in my project because of this. Can you please advise me on the simplest way to disable the Image feature in this theme? Any assistance is much appreciated. Thank you!

**Comment by jakeb91** [comment, marked as answer]
Don't use the Cloudflare plugin at-all when building with this theme, as there are Node modules that aren't compatible (not available in Cloudflare's Node environment) - that is why you're seeing this error.

To get your page working on Cloudflare Pages (if it's fine for it to be served as static content), leave the site in static mode without the Cloudflare plugin configured at all, then when Cloudflare Pages pulls the repo in will generate the static content and serve that instead.

If you have dynamic content you need to serve then it may be necessary to host elsewhere.
